Arcade/ASTEROIDS/game.ini

```diff
@@ -0,0 +1,5 @@
+title=ASTEROIDS 
+date=2018-06-28T04:41:00Z 
+author=CDR_Xavier 
+description=The unique 2D space shooter. Ammo in the bottomless box.
+license=GNL
```

Comment:
The licence needs to be changed before this is merged anyway.
Currently it says `license=GNL` when it should be `license=GPL-2.0`.

See [here](https://github.com/eried/ArduboyCollection/blob/5fe4c86cfcca04753ed3ac39dce5fc6598e3605c/Arcade/Fatsche/game.ini) and [here](https://github.com/eried/ArduboyCollection/blob/5fe4c86cfcca04753ed3ac39dce5fc6598e3605c/Arcade/Fatsche/game.ini) for comparison.